lib/util/cache/package/redis.ts

```diff
@@ -1,17 +1,18 @@
 /* istanbul ignore file */
-import { WrappedNodeRedisClient, createNodeRedisClient } from 'handy-redis';
 import { DateTime } from 'luxon';
+import { createClient } from 'redis';
 import { logger } from '../../../logger';
 
-let client: WrappedNodeRedisClient | undefined;
+let client: ReturnType<typeof createClient> | undefined;
 
 function getKey(namespace: string, key: string): string {
   return `${namespace}-${key}`;
 }
 
-export function end(): void {
+export async function end(): Promise<void> {
   try {
-    client?.nodeRedis?.end(true); // TODO: Why is this not supported by client directly? (#9714)
+    // https://github.com/redis/node-redis#disconnecting
+    await client?.quit();
```

Comment:
Or should we use `disconnect` ?

Reply:
I get that this is equivalent behaviour to before, but is there any downside to forcibly closing the connection? From the docs:

>There are two functions that disconnect a client from the Redis server. In most scenarios you should use `.quit()` to ensure that pending commands are sent to Redis before closing a connection.